neo/rawio/openephysbinaryrawio.py

```diff
@@ -141,8 +147,9 @@ def _parse_header(self):
 
                 # Special cases for stream
                 if "SYNC" in chan_id and not self.load_sync_channel:
-                    # the channel is removed from stream but not the buffer
-                    stream_id = ""
+                    # Every stream sync channel is added as its own stream
+                    stream_id = chan_id
```

Comment:
```suggestion
                    stream_id = f"{stream_name}#{chan_id}"
```
@h-mayorquin if you have multiple probes, then we probably need the stream name as well because the `chan_id` could be the same, right?

Reply:
Yes, this makes sense. 
Do we have an example like that in our testing suite? I think you are more familiar with what we have there.